tests/e2e/specs/stories-editor/media-inserter.js

```diff
@@ -68,6 +68,28 @@ describe( 'Stories Editor Screen', () => {
 		expect( src ).toContain( 'wp-content/uploads' );
 	} );
 
+	it( 'should be possible to update Background Image', async () => {
+		// Click the media selection button.
+		await page.waitForSelector( '.amp-story-media-inserter-dropdown' );
+		await page.click( '.amp-story-media-inserter-dropdown' );
```

Comment:
Looks like these two lines might deserve a little helper such as "openMediaInserter" or something like that? Seems to be used in most of the tests, sometimes more than once (like here).

Also, could we add a test which would verify that when inserting Media from the Sidebar, the text changes correctly in the Inserter as well?

Reply:
Resolved in https://github.com/ampproject/amp-wp/pull/3109/commits/968ebfd96edacebefecfcdf572a72598806248be